adapters/caaqm.js

```diff
@@ -3,10 +3,13 @@
 import { REQUEST_TIMEOUT } from '../lib/constants';
```

Comment:
When comparing to the old adapter, I'm seeing a very different number of results, are we perhaps not pulling from the same source somehow? I'm attaching below results from both the old and new adapters and the new results seem to be missing places like Indira Colony Vistar, Pali - RSPCB, Padmapukur, Howrah - WBPCB, Knowledge Park. Oddly, I also saw results in the new adapter that didn't show up in the old one.

[caaqm_old.txt](https://github.com/openaq/openaq-fetch/files/2487350/caaqm_old.txt)
[caaqm_new.txt](https://github.com/openaq/openaq-fetch/files/2487351/caaqm_new.txt)

Reply:
I'll check that tomorrow evening.

Reply:
@jflasher I believe this is cause by the following line (in both old and new adapters):

[old L47](https://github.com/openaq/openaq-fetch/pull/527/files#diff-aebb78f756b1c9e35b8f74dd4130e1d6R47)
[new L51](https://github.com/openaq/openaq-fetch/pull/527/files#diff-aebb78f756b1c9e35b8f74dd4130e1d6L51)

When I execute the two versions at the same time I have just a single station difference. If you could try to execute this in two terminals and tee the output - my test shows the following:
```bash
michal@miura:/ccc/apps/oaq-fetch-signicode⟫ diff -y <(cat ../caaqm.1.txt  | grep 'verbose' | grep '"location":' | cut -c47- | jq -c '.location' | sort | uniq -c) <(
cat ../caaqm.2.txt | grep 'verbose' | grep '"location":' | cut -c47- | jq -c '.location' | sort | uniq -c)
      6 "Ardhali Bazar, Varanasi - UPPCB"                             6 "Ardhali Bazar, Varanasi - UPPCB"
      5 "Aya Nagar, New Delhi - IMD"                                  5 "Aya Nagar, New Delhi - IMD"
      6 "Bollaram Industrial Area, Hyderabad - TSPCB"                 6 "Bollaram Industrial Area, Hyderabad - TSPCB"
      5 "BTM Layout, Bengaluru - CPCB"                                5 "BTM Layout, Bengaluru - CPCB"
      5 "Burari Crossing, New Delhi - IMD"                            5 "Burari Crossing, New Delhi - IMD"
      5 "Central School, Lucknow - CPCB"                              5 "Central School, Lucknow - CPCB"
      6 "Central University, Hyderabad - TSPCB"                       6 "Central University, Hyderabad - TSPCB"
      4 "City Railway Station, Bengaluru - KSPCB"                     4 "City Railway Station, Bengaluru - KSPCB"
      4 "Collectorate, Gaya - BSPCB"                                  4 "Collectorate, Gaya - BSPCB"
      5 "CRRI Mathura Road, New Delhi - IMD"                          5 "CRRI Mathura Road, New Delhi - IMD"
      6 "DTU, New Delhi - CPCB"                                       6 "DTU, New Delhi - CPCB"
      6 "Golden Temple, Amritsar - PPCB"                              6 "Golden Temple, Amritsar - PPCB"
      6 "GVM Corporation, Visakhapatnam - APPCB"                      6 "GVM Corporation, Visakhapatnam - APPCB"
      6 "ICRISAT Patancheru, Hyderabad - TSPCB"                       6 "ICRISAT Patancheru, Hyderabad - TSPCB"
      5 "IGSC Planetarium Complex, Patna - BSPCB"                     5 "IGSC Planetarium Complex, Patna - BSPCB"
      4 "IHBAS, Dilshad Garden,New Delhi - CPCB"                      4 "IHBAS, Dilshad Garden,New Delhi - CPCB"
      5 "Lodhi Road, New Delhi - IMD"                                 5 "Lodhi Road, New Delhi - IMD"
      4 "MD University, Rohtak, Haryana - HSPCB"                      4 "MD University, Rohtak, Haryana - HSPCB"
      6 "More Chowk Waluj, Aurangabad - MPCB"                 <
      5 "Muzaffarpur Collectorate,Muzaffarpur - BSPCB"                5 "Muzaffarpur Collectorate,Muzaffarpur - BSPCB"
      5 "NSIT Dwarka, New Delhi - CPCB"                               5 "NSIT Dwarka, New Delhi - CPCB"
      4 "Peenya, Bengaluru - CPCB"                                    4 "Peenya, Bengaluru - CPCB"
      6 "Plammoodu, Thiruvananthapuram - Kerala PCB"                  6 "Plammoodu, Thiruvananthapuram - Kerala PCB"
      5 "RIMT University, Mandi Gobindgarh - PPCB"                    5 "RIMT University, Mandi Gobindgarh - PPCB"
      4 "Sanegurava Halli, Bengaluru - KSPCB"                         4 "Sanegurava Halli, Bengaluru - KSPCB"
      6 "Sector - 125, Noida, UP - UPPCB"                             6 "Sector - 125, Noida, UP - UPPCB"
      4 "Sector- 16A, Faridabad, Haryana - HSPCB"                     4 "Sector- 16A, Faridabad, Haryana - HSPCB"
      5 "Sector - 62, Noida, UP - IMD"                                5 "Sector - 62, Noida, UP - IMD"
                                                              >       5 "Sector-6, Panchkula - HSPCB "
      5 "Shadipur, New Delhi - CPCB"                                  5 "Shadipur, New Delhi - CPCB"
      5 "Tata Stadium, Jorapokhar - JSPCB"                            5 "Tata Stadium, Jorapokhar - JSPCB"
      6 "Tirumala, Tirupati - APPCB"                                  6 "Tirumala, Tirupati - APPCB"
      6 "Vasundhara, Ghaziabad, UP - UPPCB"                           6 "Vasundhara, Ghaziabad, UP - UPPCB"
      5 "Vikas Sadan, Gurgaon, Haryana - HSPCB"                       5 "Vikas Sadan, Gurgaon, Haryana - HSPCB"
```

Reply:
Such a beautiful one line diff!